scripts/train.py

```diff
@@ -1,10 +1,9 @@
-"""
-Random experiments. As this file expands, I will probably move individual
-experiments into an scripts/ directory.
+"""Train an IRL algorithm and plot its output.
 
-These scripts are meant to be run in a Jupyter notebook (displays figures)
-but also automatically save timestamped figures to the output/ directory.
-"""
+Can be used as a CLI script, or the `train_and_plot` function can be called
```

Comment:
> or the `train_and_plot` function can be called directly

I don't think anyone outside of `scripts/train.py` can import `train_and_plot`, since `scripts/` is not installed via `pip install -e .`.

Reply:
Merging `imitation/scripts.py` into this file breaks the Jupyter notebook right now, which has an `import imitation.scripts as scripts`.